pkg/reconciler/awscognitosource/adapter.go

```diff
@@ -50,128 +43,45 @@ type adapterConfig struct {
 	configs source.ConfigAccessor
 }
 
-// reconcileAdapter reconciles the state of the source's adapter.
-func (r *Reconciler) reconcileAdapter(ctx context.Context, arn arn.ARN) error {
```

Comment:
All the type-specific `reconcileAdapter` methods are now gone, replaced by a single generic reconciler.